vanilla-tests/src/main/java/fixtures/extensibleenums/models/Pet.java

```diff
@@ -7,7 +7,7 @@
 @Fluent
 public final class Pet {
     /*
-     * The name property.
+     * name
```

Comment:
We probably need better docs here? But not critial

Reply:
I think these docs are from Swagger and not sure why it changed. I will go look at the swagger.

Reply:
I think we used to print "The {} property" when Swagger doc is missing but I don't know why this is changed either

Reply:
Yeah, if it was empty we would have generated "The {} property" but in this case swagger has non-empty doc. So, we just use that. It might be difficult to determine if the doc is insufficient from swagger.

Reply:
It appears that swagger of testservice changed.